docs/content/en/docs/triggers/message-queue-trigger-kind-keda/nats-streaming.md

```diff
@@ -13,30 +13,29 @@ You will also need NATS Streaming server setup which is reachable from the Fissi
 ## Installation
 
 If you want to setup NATS Streaming server on the Kubernetes cluster, you can use the [information here](https://github.com/nats-io/nats-streaming-server) or you can check the documentation for nats streaming [docs](https://docs.nats.io/nats-on-kubernetes/minimal-setup).  
-Also i have created a yaml file in nats-streaming-http-connector/test/nats-streaming-server folder, you can use that directly(in that i have already configured monitoring).
-
+You can also setup NATS streaming server with this [yaml](https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/nats-streaming-server/nats-dep.yaml) file.(Monitoring is already configured)
 
 {{% notice info %}}
 NATS streaming keda connector uses NATS monitoring to scale the deployment, to enable monitoring in nats we need to pass flags as below, you can get more [information here](https://docs.nats.io/nats-server/configuration/monitoring)
 
 ```bash
--m, --http_port PORT             HTTP PORT for monitoring    
--ms,--https_port PORT            Use HTTPS PORT for monitoring  
-
+-m, --http_port PORT             HTTP PORT for monitoring
+-ms,--https_port PORT            Use HTTPS PORT for monitoring
 ```
+
 {{% /notice %}}
 
 ```sh
 $ kubectl apply -f nats-dep.yaml
 NAME                                         READY   STATUS    RESTARTS   AGE
 nats-streaming-deployment-646768fcfd-qtpmk   1/1     Running   0          8s
 ```
-You can find above file reference [here](https://github.com/fission/keda-connectors/blob/master/nats-streaming-http-connector/test/nats-streaming-server/nats-dep.yaml)  
```

Comment:
This references we need to give, don't remove that

Reply:
This reference I have added in this line
You can also setup NATS streaming server with this yaml file.(Monitoring is already configured).
Should I still add here?